test/integration/002_smoke_test_scenario/3rd-party_LICENSE

```diff
@@ -0,0 +1 @@
+OpenScenario (xosc) and OpenDrive (xodr) files in this folder and sub-folders are copied from esmini (https://github.com/esmini/esmini), licensed under MPL-2.0.
```

Comment:
I would say that adding the SPDX compliant header into the files would be better. MAybe this tutorial is also a good source? https://github.com/david-a-wheeler/spdx-tutorial

Reply:
Yes I thought about it, but wasn't sure how to handle this in an xml file. Also, I wasn't sure about the copyright, since the original files do not contain a copyright notice. But I can figure something out.

Reply:
I added copyright and spdx to the files now.